inc/project.class.php

```diff
@@ -2420,7 +2420,7 @@ static function showKanban($ID) {
                ],
                'entities_id' => [
                   'type'   => 'hidden',
-                  'value'  => $_SESSION['glpiactive_entity']
+                  'value'  => $project->getField("entities_id"),
```

Comment:
In the global Kanban, value is not correct:
![image](https://user-images.githubusercontent.com/33253653/108046227-aa2b8d00-7044-11eb-9bae-7fa073ba0a74.png)

Reply:
fixed